semeio/jobs/scripts/fm_stea.py

```diff
@@ -25,14 +26,34 @@ def _get_args_parser():
         help="STEA config file, yaml format required",
         type=valid_file,
     )
+    parser.add_argument(
+        "-r",
+        "--response_file",
+        help="STEA response, json format",
+        default="stea_response.json",
+    )
     return parser
 
 
 def main_entry_point():
     parser = _get_args_parser()
     options = parser.parse_args()
     stea_input = stea.SteaInput([options.config])
-    res = stea.calculate(stea_input)
-    for res, value in res.results(stea.SteaKeys.CORPORATE).items():
+    result = stea.calculate(stea_input)
+    for res, value in result.results(stea.SteaKeys.CORPORATE).items():
         with open("{}_0".format(res), "w") as ofh:
             ofh.write("{}\n".format(value))
+    full_response = _build_full_response(stea_input, result)
+    with open(options.response_file, "w") as fout:
+        json.dump(full_response, fout, indent=4)
+
+
+def _build_full_response(stea_input, result):
+    full_response = {}
+    client = stea.SteaClient(stea_input.stea_server)
+    project = client.get_project(
+        stea_input.project_id, stea_input.project_version, stea_input.config_date
+    )
+    full_response["response"] = result.data[stea.SteaKeys.KEY_VALUES]
+    full_response["profiles"] = project.profiles
+    return full_response
```

Comment:
I find this function a bit weird.. It both takes some already generated response, and also creates a new client to get more information - and then merges. I think it would make sense to separate the functionality of returning the profile response in a single function, and have another that formats the result to something that can be dumped to a json.

Reply:
I dont disagree, but we would have to refactor `STEA` in order to achieve that, and dont really think that is within the scope.

Reply:
Thanks for splitting the functions. Yeah, we can't really do more now without starting to change things in the stea library, and I agree that is not within the scope.

Looks better though!